cccs-build/superset/Dockerfile

```
@@ -1,7 +1,7 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_11376_a25c34e1
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20240222140939_b9084
+FROM uchimera.azurecr.io/cccs/superset-base:feature_flow-10_20240304202928_b9281
```

```suggestion
FROM uchimera.azurecr.io/cccs/superset-base:feature_flow-10_20240304202928_b9281
```
FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20240222140939_b9084